### ISSUE TITLE
Suggestion: try to add an "archiving" section

### ISSUE THREAD

**Issue by ricardonevesmusic** [markusfisch/Libra#24]
Suggestion: try to add an "archiving" section

This is important to avoid having or being forced to delete old issues/problems/decisions when they're no longer relevant.

This would be nice to see what you did previously in hindsight, instead of having to delete it.

Obviously, eventually you can export and import again, but that might not be as convenient as being able to access old "items" inside an "archive" section/field.

That's it.

**Comment by ricardonevesmusic** [comment]
Update: export/import seem to not be working properly.

Tried to create one extra item, export it, delete that item and then import the previous save/export and the item didn't reappear (and eventually the app began malfunctioning/crashing for a while, until it got back to normal).

**Comment by markusfisch** [comment]
Good idea! 👍  And yes, I can totally understand that exporting and importing again is quite cumbersome and not a good solution to this. So I will add an Archive as soon as I find the time.
